migrations/migrations/20191202184912-create-user.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
```

Comment:
This also affects the model definition. Table columns must be defined using snake_case format. So in your model definition you can add the `field` option to specify the column name., for example :

```
lastName: {
        type: DataTypes.STRING,
        allowNull: false,
        field: 'last_name'
      },
```

And here in the migration use `last_name` instead of `lastName `;

Table names also use `snake_case`, so use `users` intead of `Users`.